packages/omg-js-rootchain/src/rootchain.js

```diff
@@ -126,22 +126,26 @@ class RootChain {
    * Blocks a standard exit by showing the exiting output was spent.
    *
    * @method challengeStandardExit
-   * @param {number} outputId Identifier of the exiting output.
+   * @param {number} standardExitId Identifier of the exiting output.
    * @param {string} challengeTx RLP encoded transaction that spends the exiting output.
    * @param {number} inputIndex Which input to the challenging tx corresponds to the exiting output.
    * @param {string} challengeTxSig Signature from the exiting output owner over the spend.
    * @param {Object} txOptions transaction options, such as `from`, gas` and `privateKey`
    * @return {string} transaction hash of the call
    */
-  async challengeStandardExit (outputId, challengeTx, inputIndex, challengeTxSig, txOptions) {
+  async challengeStandardExit (standardExitId, challengeTx, inputIndex, challengeTxSig, txOptions) {
```

Comment:
🎊 This is for https://github.com/omisego/omg-js/issues/89 right ?
seems like we have been seeing a lot of issues with bignum in general. Do you know why the integration tests could not catch it ?